recipes/k8/build.sh

```
@@ -25,13 +34,14 @@ patch -p0 < ${RECIPE_DIR}/nodejs-x86_64.patch
 
 # The provided configure script is a sh/python hybrid which boils down to one line of Python
 python -c "import configure" --without-node-snapshot --without-etw --without-npm --without-inspector --without-dtrace
-make -j3
+make "${THREADS}"
 popd
```

Shellcheck

[warning] Use 'popd ... || exit' or 'popd ... || return' in case popd fails.

(SC2164)